src/Middleware/RequestQueue/src/Microsoft.AspNetCore.RequestQueue.csproj

```
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core middleware for caching HTTP responses on the server.</Description>
```

```suggestion
    <Description>ASP.NET Core middleware for something?</Description>
```

Maybe? Not sure what this middleware is for, but let's update the description.

---

```suggestion
    <Description>ASP.NET Core middleware for queuing incoming HTTP requests.</Description>
```